src/selectors/metadata.js

```diff
@@ -109,6 +109,18 @@ export const mappedDefinitionsListSelector = memoize(moduleName =>
 	),
 );
 
+export const mappedCustomDefinitionsListSelector = memoize(moduleName =>
+	createSelector(mappedDefinitionsListSelector(moduleName), definitions =>
+		definitions.filter(a => a.get("isBuiltIn") === false).toJS(),
```

Comment:
please remove .toJS(), we are trying to follow rule that selectors return Map or List